src/http/cookie.cr

```diff
@@ -58,7 +58,7 @@ module HTTP
         # valid characters for cookie-name per https://tools.ietf.org/html/rfc6265#section-4.1.1
         # and https://tools.ietf.org/html/rfc2616#section-2.2
         # "!#$%&'*+-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ^_`abcdefghijklmnopqrstuvwxyz|~"
-        unless (0x21...0x7f).includes?(byte) && byte != 0x22 && byte != 0x28 && byte != 0x29 && byte != 0x2c && byte != 0x2f && !(0x3a..0x40).includes?(byte) && !(0x5b..0x5d).includes?(byte) && byte != 0x7b && byte != 0x7d
+        if !valid_cookie_value_byte?(byte) || byte.in?(0x3a..0x40) || byte.in?(0x5b..0x5d) || byte.in?(0x7b, 0x7d)
```

Comment:
Looks like `0x28` and `0x29` got lost in transformation.
Also `0x3B` and `0x5C` are excluded by the ranges here, so we could pull them out of `valid_cookie_value_byte?` because they are redundant at this call site.